### PR TITLE
[FLINK-25882] Translate updated privacy policy to Chinese

### PR DESCRIPTION
**Pull request by liyubin117**
As https://github.com/apache/flink-web/pull/503 has been merged, updated privacy policy need to be translated.

### CODE REVIEW

File: content/news/2022/01/31/release-statefun-3.2.0.html [hunk 1]
```diff
@@ -33,6 +33,8 @@
       /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
       /* We explicitly disable cookie tracking to avoid privacy issues */
       _paq.push(['disableCookies']);
+      /* Measure a visit to flink.apache.org and nightlies.apache.org/flink as the same visit */
+      _paq.push(["setDomains", ["*.flink.apache.org","*.nightlies.apache.org/flink"]]);
```
Comment (MartijnVisser) [inline content/news/2022/01/31/release-statefun-3.2.0.html:37]: I was not expecting this to appear in this PR, but it seems that you've found one page where this wasn't properly set yet when you were rebuilding the website. Thanks :)

**Review by MartijnVisser — APPROVED** [review]
Thanks a lot for providing the translation. Your rebuilding of the website is also helping fix some other issues, so thank you for that! LGTM

**Review by curcur — APPROVED** [review]
It looks good to me.

Thanks very much for the efforts. Nice translation!

File: content/news/2022/01/31/release-statefun-3.2.0.html (hunk 1 above)
Comment (liyubin117) [reply content/news/2022/01/31/release-statefun-3.2.0.html:37]: It's my pleasure

**Pull request merged**